server.js

```diff
@@ -1,33 +1,77 @@
-import express from 'express'
-import bodyParser from 'body-parser'
-import cors from 'cors'
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
-const port = process.env.PORT || 8080
-const app = express()
-
-// Add middlewares to enable cors and json body parsing
+import express from 'express';
+import bodyParser from 'body-parser';
+import cors from 'cors';
+
+//My data for the Golden Globe endpoints  
+import goldenGlobes from './data/golden-globes.json';
+
+// For no results 
+const ERROR_NOMINEES_NOT_FOUND = {error : 'No results found'}
+
+const app = express();
+ 
+// middlewares to anable cors and json nody parsing 
 app.use(cors())
 app.use(bodyParser.json())
 
-// Start defining your routes here
-app.get('/', (req, res) => {
-  res.send('Hello world')
-})
 
-// Start the server
-app.listen(port, () => {
-  console.log(`Server running on http://localhost:${port}`)
+// My routes 
+app.get('/', ( request, response) => {
+  response.send("Hello internet");
+});
+
+app.get('/goldenglobes' , (request, response) => {
+  response.json(goldenGlobes);
+});
+
+ //get Golden Globe data based on name of nominee (movie or actor)
+app.get('/goldenglobes/:name' , (request, response) => {
+  const {name} = request.params;
+  const filterdGoldenGlobes = goldenGlobes.filter(
+    goldenGlobes => goldenGlobes.nominee === name);
+
+  if(filterdGoldenGlobes.length === 0) {
+    response.status(404).json(ERROR_NOMINEES_NOT_FOUND);
+  } else {
+    response.json(filterdGoldenGlobes); 
+  }
+});
+
+
+//Checks is a specific person or film  was nominated a specific year
+//Not great, because I think actors can be nominateed in several categorys 
+app.get('/goldenglobes/:name/:year', (request, response) => {
+  const {year, name} = request.params;
+  const nominatedYear = goldenGlobes.find(
```

Comment:
I think you could use `.filter()` instead of `.find()` - first method will find all elements that fulfill those rules, second method will only find first one